Src/libCZI/libCZI_exceptions.h

```diff
@@ -138,7 +138,7 @@ namespace libCZI
     struct LibCZICZIParseException : public LibCZIException
     {
         /// Values that represent different error conditions.
-        enum class ErrorCode
+        enum class ErrorType
```

Comment:
This renaming makes total sense, however - it would be an "incompatible API-change". We'd have to bump version number to "1.0.0" (shudder), and probably it would require changes in consuming projects. So - I'd be very reluctant to make a breaking change. There better should be a good reason, and "purely cosmetic" I'd see critical. Is there really a good reason?